scripts/install_dependencies.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# TODO test on a fresh Linux install and add any missing dependencies
```

Comment:
Converted this comment to an issue:
https://github.com/pottumuusi/neotetris/issues/2

Please remove the comment.